member-service/src/main/kotlin/org/jeongmo/migration/member/application/service/MemberService.kt

```
@@ -41,4 +41,20 @@ class MemberService(
+    override fun updateMemberInfos(id: Long, request: UpdateMemberInfoRequest): UpdateMemberInfoResponse {
+        val foundMember = memberRepository.findById(id) ?: throw MemberException(MemberErrorCode.NOT_FOUND)
+        foundMember.changeNickname(request.nickname)
+        return UpdateMemberInfoResponse.fromDomain(memberRepository.save(member = foundMember))
+    }
+
+    override fun deleteMember(id: Long) {
+        try {
+            if (!memberRepository.delete(id)) {
+                throw MemberException(MemberErrorCode.ALREADY_DELETE)
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)